dli_app/mod_wiki/forms.py

```diff
@@ -91,3 +91,45 @@ def validate(self):
             ),
         ],
     )
+
+class AskQuestionForm(Form):
+
+    """A form for asking questions to administrators"""
+    def __init__(self, *args, **kwargs):
+        """Initialize a ForgotForm"""
+        Form.__init__(self, *args, **kwargs)
+
+    def validate(self):
+        """Validate the form"""
+        if not Form.validate(self):
+            return False
+        return True
+
+    email = TextField(
+        'Email',
```

Comment:
Change this to "Your Email" to make it obvious to the user what they should enter